.github/workflows/e2e_and_lighthouse_performance_tests.yml

```diff
@@ -162,7 +163,12 @@ jobs:
       - name: Install ffmpeg
         run: sudo apt install ffmpeg
       - name: Run E2E Test ${{ matrix.suite }}
-        run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --skip-install --skip-build --suite=${{ matrix.suite }} --prod_env
+        run: xvfb-run -a --server-args="-screen 0, 1285x1000x24"
```

Comment:
Is there any reason to change this? There might have been a reason previously to have all the "run" commands in a single line; I'm not sure if multi-line commands work in this script. (They may be treated as separate commands.)

Reply:
This change was suggested by vojtech. I intended to follow the previous style but since he requested those changes i did that

Reply:
Thanks -- for this and below, I have two questions. (a) Do we already do this elsewhere, and (b) does it work?

If the answer to either (a) or (b) is yes, then I'm fine leaving it as you have it now.

Reply:
okay i found some better insights into this from this [link](https://9to5answer.com/github-action-split-long-command-into-multiple-lines) . I'll amend new changes as per according to it